src/bramble/client/ProjectStats.js

```diff
@@ -0,0 +1,263 @@
+/* eslint
```

Comment:
In addition, I am not sure if you've actually reached consensus [here](https://github.com/mozilla/thimble.mozilla.org/issues/1800) about which approach to take.